python/cuml/common/base.pyx

```diff
@@ -164,16 +162,24 @@ class Base:
         del base  # optional!
     """
 
-    def __init__(self, handle=None, verbosity=logger.LEVEL_INFO,
-                 output_type=None, verbose=None):
+    def __init__(self, handle=None, verbose=True,
+                 output_type=None):
         """
         Constructor. All children must call init method of this base class.
 
         """
         self.handle = cuml.common.handle.Handle() if handle is None else handle
-        self.verbosity = verbosity
-        if verbose is not None:
-            logger.warn("'verbose' flag is deprecated in favor of 'verbosity'")
+
+        # Internally, self.verbose follows the spdlog/c++ standard of
+        # 0 is most logging, and logging decreases from there.
+        # So if the user passes an int value for logging, we convert it.
+        if verbose is True:
+            self.verbose = logger.level_debug
+        elif verbose is False:
+            self.verbose = logger.level_info
+        else:
+            # Using max in case user gives a verbosity value greater than 6
+            self.verbose = max(6 - verbose, 0)
```

Comment:
Just a comment. I think for this PR it is fine to do this reversal. However, ideally, this should be taken care of in the C++ layer itself to avoid confusion between these 2 layer interfaces

Reply:
yeah, agreed, will open a github issue to make this consistent

Reply:
Thanks